package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "redux-helpers",
+  "version": "",
+  "contributors": [
+    "The Polymer Authors"
+  ],
+  "license": "https://polymer.github.io/LICENSE.txt",
+  "dependencies": {
+    "@0xcda7a/redux-es6": "^3.7.2",
```

Comment:
devDependency?

Reply:
yup, now that i've moved `combineReducers` away, good call.